tests/utils/test_Misc.py

```diff
@@ -259,4 +261,68 @@ def test_under_debugger(self):
         if 'pydevd' in sys.modules:
             assert under_debugger() is True
         else:
-            assert under_debugger() is False
\ No newline at end of file
+            assert under_debugger() is False
+
+    def test_base64_to_str_and_str_to_base64(self):
+        text = "Lorem Ipsum AAAAAA"
+        base64_encoded_string = base64.b64encode(text.encode()).decode()
+        assert base64_to_str(base64_encoded_string) == text
+        assert str_to_base64(text                 ) == base64_encoded_string
+
+    def test_env_value(self):
+        assert env_value("ENV_VAR_1") == "ENV_VAR_1_VALUE"
+
+    def test_env_vars(self):
+        os.environ.__setitem__("ENV_VAR_FROM_CODE", "ENV_VAR_FROM_CODE_VALUE")
+        loaded_env_vars = env_vars()
+        assert loaded_env_vars.get("ENV_VAR_1")         == 'ENV_VAR_1_VALUE'
+        assert loaded_env_vars.get("ENV_VAR_2")         == 'ENV_VAR_2_VALUE'
+        assert loaded_env_vars.get("ENV_VAR_FROM_CODE") == 'ENV_VAR_FROM_CODE_VALUE'
```

Comment:
almost there with the formating :)

align on )

Reply:
fixed